manager/role_manager.go

```diff
@@ -24,29 +26,47 @@ type roleManager struct {
 	// pending contains changed nodes that have not yet been reconciled in
 	// the raft member list.
 	pending map[string]*api.Node
+
+	// removed contains raft members, indexed by node ID, whose nodes have since
+	// been deleted - these members need to be removed from raft
+	removed map[string]*api.RaftMember
```

Comment:
maybe rename to to_be_removed to clarify the meaning here ?

Reply:
If it were interrupted, we'd have to try again in another tick, so wouldn't it have to be persisted?